Duplicati/UnitTest/BasicSetupHelper.cs

```diff
@@ -211,6 +212,17 @@ protected static void ZipFileExtractToDirectory(string sourceArchiveFileName, st
                 ZipFile.ExtractToDirectory(sourceArchiveFileName, destinationDirectoryName);
             }
         }
+
+        /// <summary>
+        /// Write file <paramref name="path"/> with <paramref name="contents"/>.
+        /// </summary>
+        protected static void WriteFile(string path, byte[] contents)

```

Comment:
This could be useful in other tests.  Can we move this to the `TestUtils` class?

Reply:
Addressed in latest commit.